app/src/utils/itemCount.ts

```diff
@@ -0,0 +1,23 @@
+import { apiResponse } from "./api";
+
+/**
+ * Returns a map of collection uuids with corresponding item counts.
+ * @param {string[]} uuids - collection uuids
+ */
+export const getItemsCountFromCollections = async (uuids: string[]) => {
+  const results = {};
+  const apiUrl = `${process.env.API_URL}/api/v2/collections/`;
+  await Promise.all(
+    uuids.map(async (uuid) => {
```

Comment:
This is me being nitpicky but the syntax is interesting. This works so no need to change but I would generally prefer to create the list of unresolved promises in a variable and then pass it to the `Promise.all` call. Just minor readability comment.

Reply:
Is this what you're thinking ?